libyul/backends/evm/OptimizedEVMCodeTransform.h

```diff
@@ -116,6 +119,7 @@ class OptimizedEVMCodeTransform
 	std::vector<StackTooDeepError> m_stackErrors;
 	/// True if it simulates functions with jumps. False otherwise. True for legacy bytecode
 	bool m_simulateFunctionsWithJumps = true;
+	size_t const m_reachableStackDepth = 16;
```

Comment:
At first I wanted to ask why this is hard-coded to 16 even on EOF, but looks like the value here is irrelevant because the constructor overwrites it. I think it's much less confusing if you use default initialization to hide that unused value:

```suggestion
	size_t const m_reachableStackDepth{};
```

After all it does not matter what it's initialized with here, only that it's initialized.

Same in `StackLayoutGenerator` and `StackShufflingTest`.